src/kernel/include/thd.h

```diff
@@ -153,6 +166,17 @@ thd_rcvcap_sched(struct thread *t)
 }
 
 static void
+thd_next_thdinfo_update(struct cos_cpu_local_info *cli, struct thread *thd, 
+			struct tcap *tc, tcap_prio_t prio)
+{
+	struct next_thdinfo *nti = (struct next_thdinfo *)cli->next_ti;
```

Comment:
Oh yes, I didn't think in that perspective, I'm sorry. I was thinking in terms of size of `cos_cpu_local_info`. I will change it as you suggested. Thank you.